Resources/Prototypes/DeltaV/Entities/Clothing/Shoes/specific.yml

```
@@ -0,0 +1,14 @@
+- type: entity  #Delta-V realistic chemistry shoes
```

```suggestion
- type: entity
```
No need for a change marker since it is in a DeltaV file.